### ISSUE TITLE
hadolint JSON schema for IDE integration is outdated

### ISSUE THREAD

**Issue by Mazorius** [hadolint/hadolint#601]
- [x] This is a bug report
- [x] I searched existing issues before opening this one

I wanted to write a hadolint configuration file in an Jetbrains IDE.
For that I imported the https://json.schemastore.org/hadolint.json schema to have autocompletion and validation.

### Expected behavior

The configuration file works well.

### Actual behavior

The configuration is outdated.

### Steps to reproduce the behavior

Use a Jetbrains IDE and configure the a custom JSON schema: https://www.jetbrains.com/help/idea/json.html?keymap=secondary_macos_system_shortcuts#f502d790

<img width="938" alt="Screen Shot 2021-04-13 at 11 34 40" src="https://user-images.githubusercontent.com/11423826/114531521-4391ac80-9c4c-11eb-92b5-41d671e8bd50.png">

Create `.hadolint.yaml` file and add the following:

``` yaml
---
trustedRegistries:
  - docker.example.com

strict-labels: true

label-schema:
  maintainer: text
  version: semver
  documentation: url
  git-revision: hash
  license: spdx

```

It should look like:

<img width="733" alt="Screen Shot 2021-04-13 at 11 36 17" src="https://user-images.githubusercontent.com/11423826/114531767-7cca1c80-9c4c-11eb-9874-465de30c0d2d.png">

**Additional environment details (OS, stack version, etc.)**

- macOS
- IntelliJ IDEA Ultimate
- v2.1.0
